packages/ckeditor5-image/tests/imagestyle/imagestyleui.js

```diff
@@ -240,14 +240,18 @@ describe( 'ImageStyleUI', () => {
 		it( 'should define the drop-down properties and children properly', () => {
 			for ( const { config, view, buttonView } of dropdowns ) {
 				const defaultItem = allStyles.find( style => style.name === config.defaultItem.replace( 'imageStyle:', '' ) );
+				const expectedLabel = ( config.title ? `${ config.title }: ` : '' ) + defaultItem.title;
 
 				expect( view ).to.be.instanceOf( DropdownView );
 				expect( buttonView ).to.be.instanceOf( SplitButtonView );
 
-				expect( buttonView.label ).to.equal( ( config.title ? `${ config.title }: ` : '' ) + defaultItem.title );
+				expect( buttonView.label ).to.equal( expectedLabel );
 				expect( buttonView.tooltip ).to.be.true;
 				expect( buttonView.class ).to.be.null;
 
+				expect( buttonView.arrowView.label ).to.equal( expectedLabel );
+				expect( buttonView.arrowView.tooltip ).to.be.true;
```

Comment:
Obsolete per my comment in [codeblockui.js](https://github.com/ckeditor/ckeditor5/pull/11859/files?show-viewed-files=true&file-filters%5B%5D=#diff-a75a67319b38c5dc7c27b30eeaff970892cdd73e01b28920a569d161458ea8c7).